livre/core/Render/GLContext.h

```diff
@@ -43,6 +43,8 @@ class GLContext : public boost::enable_shared_from_this< GLContext >
      */
     void shareContext( GLContextPtr dstContextPtr );
 
+    const GLEWContext* glewGetContext() const;
```

Comment:
On Windows you cannot be clean of GLEW when using any OpenGL >1.1 function, unless you want to write your own glew.